### PR TITLE
Share recitation page

### PR DESCRIPTION
**Pull request by muhajirdev**
### Summary
image

### CODE REVIEW

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**…reciterId\]/\[chapterId\]**](https://quran-com-d0zk4ouxa-qurancom.vercel.app/reciters/[reciterId]/[chapterId]) | [**…erId\]/chapterId.module**](https://quran-com-d0zk4ouxa-qurancom.vercel.app/reciters/[reciterId]/chapterId.module) |
|:-:|:-:|
|<a href="https://quran-com-d0zk4ouxa-qurancom.vercel.app/reciters/[reciterId]/[chapterId]">Screenshot of /reciters/[reciterId]/[chapterId]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/JjinHl6lqwFWCtZZyRz-7rKgA4IOnj1s1li42vMcJOG3xBdJ_OAHEq4WTlZgKXBh-2J-4czPcXUeT_S812ei2A.png">(view full size)</a> |<a href="https://quran-com-d0zk4ouxa-qurancom.vercel.app/reciters/[reciterId]/chapterId.module">Screenshot of /reciters/[reciterId]/chapterId.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/F2pZLhaCPvsz1JFcdiGbLaI-ACQxHVY_prmzQ1XfNWJUI76HO7W1FbzelGta8Uxe3TICm9WCJUhB6YQ-EVWIyw.png">(view full size)</a> |



| [**/reciters/\[reciterId\]**](https://quran-com-d0zk4ouxa-qurancom.vercel.app/reciters/[reciterId]) |
|:-:|
|<a href="https://quran-com-d0zk4ouxa-qurancom.vercel.app/reciters/[reciterId]">Screenshot of /reciters/[reciterId]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/bK5uiB8ZNVgJ4wbxKUFsQd6uuTASMYZlYkxajOfJqc0hm-pd7LtT4YUTJeaHH5e-E5ZRvQefOUOyI_qxG0qwtg.png">(view full size)</a> |







Commit 805d86760aedb8138f84bd9f7cf320dcc6798c79 (https://quran-com-d0zk4ouxa-qurancom.vercel.app).

File: src/pages/reciters/[reciterId]/[chapterId].tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import { getSurahNavigationUrl } from 'src/utils/navigation';
+import { getCurrentPath } from 'src/utils/url';
+import Chapter from 'types/Chapter';
+import Reciter from 'types/Reciter';
+
+type ShareRecitationPageProps = {
+  selectedReciter: Reciter;
+  selectedChapter: Chapter;
+};
+const ShareRecitationPage = ({ selectedReciter, selectedChapter }: ShareRecitationPageProps) => {
```
Comment (muhajirdev) [inline src/pages/reciters/[reciterId]/[chapterId].tsx:36]: RecitationPage? SharedRecitationPage?

File: i18n.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "/apps": ["apps"],
     "/radio": ["home", "quick-links", "radio"],
     "/reciters/[reciterId]": ["home", "quick-links", "radio", "reciter"],
-    "/reciters": [ "reciter", "home", "quick-links"]
+    "/reciters/[reciterId]/[chapterId]": ["home", "quick-links", "radio", "reciter"],
```
Comment (tifa2UP) [inline i18n.json:26]: Do we need all these strings?

File: src/pages/reciters/[reciterId]/[chapterId].tsx (hunk 1 above)
Comment (tifa2UP) [reply src/pages/reciters/[reciterId]/[chapterId].tsx:36]: RecitationPage is good

File: src/pages/reciters/[reciterId]/[chapterId].tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      playFrom({
+        chapterId: Number(selectedChapter.id),
+        reciterId: selectedReciter.id,
+        timestamp: 0,
+      }),
+    );
+  };
+
+  const onCopyLinkClicked = () => {
+    logButtonClick('share-recitation__copy-link');
```
Comment (tifa2UP) [inline src/pages/reciters/[reciterId]/[chapterId].tsx:53]: is __ intended / consistent with the rest of the loggin?

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**